src/app.js

```diff
@@ -1 +1,37 @@
-// write code here
+const fs = require('fs/promises');
+const path = require('path');
+
+async function moveFile() {
+  const [source, destination] = process.argv.slice(2);
+
+  if (!source || !destination) {
+    // eslint-disable-next-line no-console
+    console.log(!source);
```

Comment:
The use of `console.log(!source)` seems unnecessary. This will log `true` if `source` is not provided, which might not be useful for debugging or user feedback.